packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -845,7 +845,7 @@ export class SpecBuilder {
     }
     if (query.filters) {
       spec.filters = query.filters.map((filter: Filter) => {
-        if (filter instanceof RelationFilter) {
```

Comment:
Can you update the comment on line 1504 (`This currently only supports relation filters (<, <=, ==, >=, >)`)?  Maybe the comment can be completely removed (I think we accidentally support nan / null filters now...)